packages/database/src/migrations/20210316222124-AddLoasEocMeaslesDashboardGroups-modifies-data.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import { arrayToDbString, insertObject } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboardGroups = [
+  {
+    organisationLevel: 'SubDistrict',
+    userGroup: 'Laos EOC User',
+    organisationUnitCode: 'LA',
+    dashboardReports: '{}',
+    name: 'Measles',
+    code: 'LAOS_EOC_Measles_Sub_District',
```

Comment:
district or sub district ?

Reply:
Laos District which is a tupaia sub_district - assuming same as this table: https://github.com/beyondessential/tupaia-backlog/issues/2361#issuecomment-799002833